help/destinations/catalog/crm/salesforce.md

```diff
@@ -122,11 +135,11 @@ If the details provided are valid, the UI displays a **Connected** status with a
 ### Fill in destination details {#destination-details}
 
 To configure details for the destination, fill in the required and optional fields below. An asterisk next to a field in the UI indicates that the field is required.
-![Sample screenshot showing how to fill in details for Salesforce CRM](../../assets/catalog/crm/salesforce/destination-details.png)
+![Platform UI screenshot showing the destination details.](../../assets/catalog/crm/salesforce/destination-details.png)
 
 *  **[!UICONTROL Name]**: A name by which you will recognize this destination in the future.
 *  **[!UICONTROL Description]**: A description that will help you identify this destination in the future.
-*  **[!UICONTROL Custom Domain]**: Your Salesforce domain.
+*  **[!UICONTROL Salesforce ID Type]**: If the identities within your Platform are of type *Contact* select **[!UICONTROL Contact]**. If the identities within your Platform are of type *Lead* select **[!UICONTROL Lead]**.
```

Comment:
Minor rewording suggestion. Can you please rephrase to be similar to my suggestion for Line: 

https://github.com/AdobeDocs/experience-platform.en/pull/118#discussion_r975381790